packages/shared/package.json

```diff
@@ -59,6 +59,7 @@
 		"knex-schema-inspector": "2.0.4",
 		"lodash": "4.17.21",
 		"micromustache": "^8.0.3",
+		"nanoid": "^4.0.0",
```

Comment:
do we want to change nanoid to v3 similar to how current API and App uses? As v4 moved to ESM only (of course we have the ESM move soon). Not sure what difference would this make though tbh.

Reply:
I think it's fine for now. It's only used in dependencies going to the app, which is already ESM. If you import that same composable in the API, it will fail, but that will resolve itself with the move to ESM 👍🏻